Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -70,6 +70,20 @@ configuration:
   name: add_comment_if_empty
   required: false
   type: 8
+- additionalinfo: 'CIDRs with prefix size smaller than this value will not be included'
+  defaultvalue: 8
+  display: 'Maximum CIDR prefix size'
+  hidden: false
+  name: maximum_cidr_size
+  required: true
+  type: 0
+- additionalinfo: 'Configure to disable top level domain in the list.'
+  defaultvalue: 'false'
+  display: 'Disable insertion top level domains'
```

Comment:
This display is a bit too technical. The user doesn't think of EDL as if it inserts results into a list. Perhaps `Drop top level domain globs`?